doc/examples-getting-started.rst

```diff
@@ -1,8 +1,8 @@
 ##############################
-Simple circuits and algorithms
+Getting started
```

Comment:
These are already sentence case. So that's probably right, but are you sure we don't want title case?

Reply:
I was considering it, but I think that's a story for another day, when we sweep over every example